### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by justin-john**
Just a link update.

### CODE REVIEW

**Comment by justin-john** [comment]
@googlebot I signed it!

File: README.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ _**Note:** If you're not using npm, you can still load `web-vitals` via `<script
 
 There are two different versions of the `web-vitals` library (the "standard" version and the "base+polyfill" version), and how you load the library depends on which version you want to use.
 
-For details on the difference between the two versions, see <a href="#which-version-is-right-for-you">which version is right for you</a>.
+For details on the difference between the two versions, see <a href="#which-bundle-is-right-for-you">which version is right for you</a>.
```
Comment (philipwalton) [inline README.md:58]:
```suggestion
For details on the difference between the two versions, see <a href="#which-bundle-is-right-for-you">which bundle is right for you</a>.
```

Comment (philipwalton) [reply README.md:58]: Good catch! Can you also change this as well?

**Review by philipwalton — CHANGES_REQUESTED** [review]

**Review by philipwalton — APPROVED** [review]

**Comment by philipwalton** [comment]
Thanks!

**Pull request merged**